getflywheel.com.arecord.json

```diff
@@ -0,0 +1,18 @@
+{
+   "providerId":"getflywheel.com",
+   "providerName":"Flywheel",
+   "serviceId":"arecord",
+   "serviceName":"Delightfully Managed WordPress Hosting",
```

Comment:
Pls. reconsider shorter name. Such a long one may be cut by some UIs (like IONOS' one).

Reply:
Update made and new commit pushed! 👍